### PR TITLE
Python api docs

### PR DESCRIPTION
**Pull request by jkbradley**
* re-ran sphinx-quickstart to get Python API doc gen working
* Changed GraphFrames version to 0.1 and release version to 0.1.0.
* documentation updates

### CODE REVIEW

File: python/docs/conf.py [hunk 1]
```diff
@@ -47,17 +47,17 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'GraphFrames'
-copyright = u''
+project = u'pygraphframes'
+copyright = u'2016, Author'
```
Comment (thunterdb) [inline python/docs/conf.py]: 'Joseph & Tim' :-)

File: python/docs/conf.py [hunk 2]
```diff
@@ -272,7 +272,7 @@
 epub_title = u'pygraphframes'
 epub_author = u'Author'
 epub_publisher = u'Author'
-epub_copyright = u'2014, Author'
+epub_copyright = u'2016, Author'
```
Comment (thunterdb) [inline python/docs/conf.py]: Same thing here

File: python/docs/Makefile [hunk 3]
```diff
@@ -7,7 +7,7 @@ SPHINXBUILD   = sphinx-build
 PAPER         =
 BUILDDIR      = _build
 
-export PYTHONPATH=$(realpath ..):$(realpath $(SPARK_HOME)/python)
+export PYTHONPATH=$(realpath ..):$(realpath $(SPARK_HOME))/python
```
Comment (thunterdb) [inline python/docs/Makefile]: change this line to `export PYTHONPATH=$(realpath ..):$(realpath $(SPARK_HOME))/python:$(realpath $(SPARK_HOME))/python/lib/py4j-0.8.2.1-src.zip`

File: python/docs/conf.py (hunk 1 above)
Comment (jkbradley) [inline python/docs/conf.py]: But also Ankur, Feynman, Xiangrui, and hopefully others in the future.  I'd prefer to leave it generic.

Comment (thunterdb) [inline python/docs/conf.py]: Then I think it is more customary to put something like "2016, the
GraphFrames project authors" and list the contributors in the README. This
is a nit, though, we can change it later.

**Comment by jkbradley** [comment]
I'll merge this now & will keep working on some doc and API tweaks.

**Pull request merged**